test_configs/sockets/all.test

```diff
@@ -0,0 +1,102 @@
+#: "Suite of tests for the sockets provider"
+{
+	prov_name: sockets,
+	test_type: [
+		FT_TEST_LATENCY,
+		FT_TEST_BANDWIDTH,
+	],
+	class_function: [
+		FT_FUNC_SEND,
+		FT_FUNC_SENDV,
+		FT_FUNC_SENDMSG,
+		FT_FUNC_INJECT,
+		FT_FUNC_INJECTDATA,
+	],
+	ep_type: [
+		FI_EP_MSG,
+		FI_EP_DGRAM,
+		FI_EP_RDM
+	],
+	av_type: [
+		FI_AV_TABLE,
+		FI_AV_MAP
+	],
+	comp_type: [
+		FT_COMP_QUEUE
```

Comment:
Ok, I notice the RMA flags are also missing. Looks like those are not supported yet.